tests/containers/bci_prepare.pm

```diff
@@ -73,12 +82,28 @@ sub run {
 
     record_info('Install', 'Install needed packages');
     my @packages = packages_to_install();
-    foreach my $pkg (@packages) {
-        record_info('pkg', $pkg);
-        zypper_call("--quiet in $pkg", timeout => 300);
+    if ($host_distri eq 'ubuntu') {
+        assert_script_run("apt-get update", timeout => 600);
+        foreach my $pkg (@packages) {
+            assert_script_run("apt-get -y install $pkg", timeout => 300);
+        }
+        assert_script_run('pip3 --quiet install --upgrade pip', timeout => 600);
+        assert_script_run("pip3 --quiet install tox", timeout => 600);
+    } elsif ($host_distri eq 'centos') {
+        assert_script_run("yum update -y --allowerasing", timeout => 600);
+        foreach my $pkg (@packages) {
+            assert_script_run("yum install -y $pkg", timeout => 300);
+        }
+        assert_script_run('pip3 --quiet install --upgrade pip', timeout => 600);
```

Comment:
those lines are repeated. What about take them off to a subroutine?

Reply:
Since the pip upgrade for sles is different, I don't think we gain a lot creating a subroutine for 2 lines repeated only twice. I would leave it as it is. If later on we add another host and this repeats 3 times, yes.

Reply:
ok now i am curious what difference `pip3.6` makes. it shouldnt. right? and usually the installation gives you symlinks to the current `pip`, so `pip3` should be enough or..?

Reply:
There was a reason, it was done some months ago when I enabled BCI tests on SLEs... but I don't remember :D 
I'll try to figure out to give you a proper answer :)